tests/helm/goldenfile_test.go

```diff
@@ -132,11 +133,172 @@ func runGoldenFileTest(t *testing.T, valuesFileName string, outputFileName strin
 // expected templates
 func fixupRenderedYaml(yaml string, chartVersion string) string {
 	checksumRegex := regexp.MustCompile("checksum/config: [a-z0-9]{64}")
+	patternString := fmt.Sprintf(
+		`(?m)^(\s*app\.kubernetes\.io\/version:\s*"%s"\s*)$`+
+			`|^(\s*chart:\s*"sumologic-%s"\s*)$`+
+			`|^(\s*chart:\s*sumologic-%s\s*)$`+
+			`|^(\s*client:\s*k8s_%s\s*)$`+
+			`|^(\s*value:\s*"%s"\s*)$`,
+		chartVersion, chartVersion, chartVersion, chartVersion, chartVersion)
+	pattern := regexp.MustCompile(patternString)
```

Comment:
As it's correct, it's not so easy to understand the regex. I would recommend to create lis of regexes, and then iterate over them and replace in the output, or even just replace bare strings. I would want to maintain it in this form after all

Reply:
addressed